components/gitpod-protocol/src/permission.ts

```diff
@@ -6,18 +6,13 @@
 
 // see below for explanation
 export const Permissions = {
-    monitor: undefined,
```

Comment:
This was used to grant access to monitor.gitpod.io, which does not exist anymore for years already.